app/views/external_links/_all.html.erb

```diff
@@ -1,9 +1,6 @@
 <% psu_digital_collections_links = document.psu_digital_collections_links
-   access_online_links = document.access_online_links
-   hathi_links = document.hathi_links %>
+   access_online_links = document.access_online_links %>
 
 <%= render partial: 'external_links/psu_digital_collections_links', locals: { psu_digital_collections_links: psu_digital_collections_links } if psu_digital_collections_links.present? %>
 
 <%= render partial: 'external_links/access_online_links', locals: { access_online_links: access_online_links } if access_online_links.present? %>
-
-<%= render partial: 'external_links/hathi_links', locals: { hathi_links: document.hathi_links } if hathi_links %>
```

Comment:
@nicktiberi Since we still have the rest of the hathi feature in the code, instead of removing these, can we use the check we used here https://github.com/psu-libraries/psulib_blacklight/blob/main/app/views/catalog/_show_availability.html.erb#L8?

Reply:
@banukutlu - if I add the check for `@document.hathi_links.dig(:etas_item)`, will that fail now for everything because ETAS is over?

If so, then the Hathi box will never appear, and I'm not sure if that's what Ruth wants. My understanding was she wanted to keep the Hathi box for the individual item view, but remove the title, and for search results view, remove the box entirely.

Sorry for any confusion on this!

Reply:
> @banukutlu - if I add the check for `@document.hathi_links.dig(:etas_item)`, will that fail now for everything because ETAS is over?

Yes, it won't find any etas item when Hathi ETAS set to false.

> 
> If so, then the Hathi box will never I, and I'm not sure if that's what Ruth wants. My understanding was she wanted to keep the Hathi box for the individual item view, but remove the title, and for search results view, remove the box entirely.
> 
> Sorry for any confusion on this!

Your understanding is right, with this change I do not mean to change the rules, just wanted to keep things working same to be able to set back if Hathi ETAS needed to be set back to true.